Automattic/jetpack
# Update used but not minimum node version

#### Changes proposed in this Pull Request:

We try to keep `.nvmrc` in sync with Calypso, and they just updated
their version. On the other hand, there don't seem to be any new
features over 16.7.0 for us to depend on, so we may as well leave the
minimum version there.

#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* If you use nvm, it should want 16.11.1 now. But if you run with 16.7.0, pnpm shouldn't complain about it.
